.github/workflows/pr-build.yaml

```diff
@@ -0,0 +1,49 @@
+name: PR Build & Test
+
+on:
+  pull_request:
+  push:
+    paths-ignore:
+      - '.github/ISSUE_TEMPLATE/**'
+    branches:
+      - main
+
+env:
+  RUSTFLAGS: -Dwarnings
+
+jobs:
+  check:
+    runs-on: windows-2019
+
+    strategy:
+      matrix:
+          # Apparently ARM targets aren't usable in GH actionsd
```

Comment:
You can cross compile, but this changes the toolchain that is doing the building and thus obviously has to match the runner's architecture.

Reply:
Here's an example of cross-compiling: https://github.com/microsoft/windows-rs/blob/master/.github/workflows/cross.yml

I don't think you really need that here but it's an option.